reana_workflow_controller/workflow_state.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""Workflow status transition management."""
+
+from reana_db.models import WorkflowStatus
+
+
+def can_transition(workflow, next_status):
```

Comment:
Looks good! What about moving this function to the Workflow model? So we can do `workflow.can_transition_to(next_status)`

Reply:
https://github.com/reanahub/reana-db/pull/63